scripts/BillsHouse.asm

```diff
@@ -95,73 +95,73 @@ MovementData_1e807:
 	db NPC_MOVEMENT_DOWN
 	db -1 ; end
 
-BillsHouseScript4:
+BillsHouseCleanupScript:
 	ld a, [wd730]
 	bit 0, a
 	ret nz
 	xor a
 	ld [wJoyIgnore], a
 	SetEvent EVENT_MET_BILL_2 ; this event seems redundant
 	SetEvent EVENT_MET_BILL
-	ld a, $0
+	ld a, SCRIPT_BILLSHOUSE_DEFAULT
 	ld [wBillsHouseCurScript], a
 	ret
 
-BillsHouseScript5:
-	ld a, $4
+BillsHousePCScript:
+	ld a, TEXT_ACTIVATE_PC
 	ldh [hSpriteIndexOrTextID], a
 	call DisplayTextID
-	ld a, $0
+	ld a, SCRIPT_BILLSHOUSE_DEFAULT
 	ld [wBillsHouseCurScript], a
 	ret
 
-BillsHouse_TextPointers:
-	dw BillsHouseText1
-	dw BillsHouseText2
-	dw BillsHouseText3
-	dw BillsHouseText4
+	def_text_pointers BillsHouse
+	dw_const BillsHouseBillIsAPokemonScript,            TEXT_BILL_IS_A_POKEMON
+	dw_const BillsHouseBillSSAnneScript,                TEXT_BILL_SS_ANNE
+	dw_const BillsHouseBillCheckOutMyRarePokemonScript, TEXT_BILL_CHECK_OUT_MY_RARE_POKEMON
+	dw_const BillsHouseActivatePCScript,                TEXT_ACTIVATE_PC
```

Comment:
`TEXT_ACTIVATE_PC` might not be unique enough considering the remaining maps.

Reply:
Also `TEXT_BILL_SS_ANNE` could be `TEXT_BILL_SS_TICKET`, so it doesn't sound related to the SS Anne map.